bodybuilder.d.ts

```diff
@@ -8,6 +8,12 @@ declare namespace bodybuilder {
 		agg: QuerySubFilterBuilder
 	) => QuerySubFilterBuilder;
 
+	export interface AggOptions {
+		name?: string;
+		meta?: object;
```

Comment:
I believe this is a typo and should be `_meta`

Reply:
Just pushed, thanks for catching!